TablesCreation

```diff
@@ -0,0 +1,112 @@
+CREATE TABLE [dbo].[Technicians](
```

Comment:
Don't want all these in the same place.  I'd like them broken down exactly like I have in this pull request: https://github.com/Rwpulley/Portfolio/pull/1  There should be a folder for CleanGreen and then another folder inside of that for tables, and then you break up each table create script into it's own file.  We don't want to just jam everything into one file, that's not a great design strategy.